github-git/3-fork-repo.md

```diff
@@ -0,0 +1,100 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+(fork-repository)=
+# How to Fork a GitHub Repository 
+
+
+:::{admonition} What you will learn:
+:class: tip
+
+In this lesson, you will learn how to <kbd>fork</kbd> (or create a copy of) a GitHub repo into your own **GitHub.com** account. You can practice forking the pyOpenSci example repository. 
+::: 
+
+:::{admonition} Activity: Fork a repository and modify a file
+
+**1. Fork the pyOpenSci practice GitHub repository**
+*******
+
+Fork the [pyOpenSci example repo](https://github.com/pyOpenSci/pyos-demo-package-contribute). Remember that a fork is a copy of a repository that is owned by someone else or an organization that lives in your GitHub account.
+
+**2. Open the file that you proposed changes to in the issue you selected in the [how to identify an issue lesson](2-identify-issue).**
```

Comment:
This point seems out of context of the rest of the page, and more like the starting point of edit-commit-files.